integrations/larky/input/query_string/inbound-query-string.yml

```diff
@@ -0,0 +1,59 @@
+data:
+  - attributes:
+      created_at: '2021-04-26T09:55:37'
+      destination_override_endpoint: 'https://echo.apps.verygood.systems'
+      entries:
+        - classifiers: {}
+          config:
+            condition: AND
+            rules:
+              - condition: null
+                expression:
+                  field: PathInfo
+                  operator: begins_with
+                  type: string
+                  values:
+                    - /post
+          id: 3e4ca047-23e2-4397-9a67-d673cedf4cc8
+          id_selector: null
+          operation: REDACT
+          operations:
+            - name: github.com/verygoodsecurity/common/compute/larky/http/Process
+              parameters:
+                script: |-
+                  load("@stdlib//builtins", builtins="builtins")
+                  load("@stdlib//json", json="json")
+                  load("@vgs//vault", "vault")
+                  load("@stdlib//urllib/parse", parse="parse")
+
+                  def process(input, ctx):
+                      query = input.query_string
+                      
+                      body = dict(parse.parse_qsl(query))
+                      
+                      body['account'] = vault.reveal(body['account']) # aka Reveal
+
+                      input.body = builtins.bytes(json.dumps(body))
```

Comment:
This is taking the query string and assigning it to the body whichnwill fail for GET requests. Let's have an example of overwriting the query string instead.

Reply:
Doesn't look like we have here an opposite method for parse_qsl()
https://github.com/verygoodsecurity/starlarky/blob/master/larky/src/main/resources/stdlib/urllib/parse.star

so came up with FOR loop